modules/core/src/main/java/org/apache/ignite/mxbean/DataRegionMetricsMXBean.java

```diff
@@ -124,6 +124,26 @@
     @MXBeanDescription("Offheap used size in bytes.")
     @Override public long getOffheapUsedSize();
 
+    /** {@inheritDoc} */
+    @MXBeanDescription("Gets memory page size.")
+    @Override public int getPageSize();
+
+    /** {@inheritDoc} */
+    @MXBeanDescription("Gets total size of pages loaded to the RAM. When persistence is disabled, this metric is equal TotalAllocatedSize.")
```

Comment:
_The_ total size of pages loaded to the RAM. When persistence is disabled, this metric is equal _to_ TotalAllocatedSize.